src/components/tasks/card/index.tsx

```diff
@@ -704,12 +694,17 @@ const Card: FC<Props> = ({
                                             tabIndex={0}
                                         />
 
-                                        {showSuggestion && (
-                                            <SuggestionBox
-                                                suggestions={suggestions}
-                                                onClickName={handleClick}
-                                                loading={isLoadingSuggestions}
-                                            />
+                                        {isLoadingSuggestions ? (
+                                            <Loader />
+                                        ) : (
+                                            showSuggestion && (
+                                                <SuggestionBox
+                                                    suggestions={suggestions}
+                                                    onSelectAssignee={
+                                                        handleClick
+                                                    }
+                                                />
+                                            )
```

Comment:
here also please if else condition

Reply:
and as per @bharati-21 comments yes this needs to be made a separate component so please make it a separate component.

Reply:
https://stackoverflow.com/questions/44046037/if-else-statement-inside-jsx-reactjs @Pratiyushkumar 
And Bharti's query is resolved. Thank you.